### DISCUSSION TITLE
Easier environment switching

### DISCUSSION THREAD

**Discussion by BoscoDomingo** [darrenburns/posting#97] [Ideas]
Right now you either restart posting loading a different .env file, or you change the .env file before the request so the new value for a variable is used.

Is there a chance to implement a simple dropdown (like the methods one) to switch environments (AKA load different .env files)? This what most API call software does (Postman, Insomnia, Bruno, etc)
![image](https://github.com/user-attachments/assets/c2c40315-87e4-48aa-8198-eec5f5481101)

**Comment by darrenburns** [comment]
Yes, I'd like to add something like this. I need to think of a nice approach though because you can load multiple `.env` files to allow for inheritance.
